### ISSUE TITLE
Cocoapods release for swift 2.3 branch

### ISSUE THREAD

**Issue by lilac** [hyperoslo/ImagePicker#234]
I noticed that there is a branch for swift 2.3 support, but there is not a release targets that. Could you release a version that is compatible with swift 2.3?

**Comment by zenangst** [comment]
Hey @lilac, we decided to jump directly to Swift 3.0 for all of our libraries, as Apple will drop support `2.x` in the near future. Hence making double releases and support both are not really in our interest. We have a `swift-2.3` branch that can be used for apps that use Swift versions prior to `3.0`.

Our decision is partly based on the Xcode release notes that state:

> Xcode 8.2 is the last release that will support Swift 2.3.  Please migrate your projects to Swift 2.3 code to Swift 3 syntax by opening the project and choosing Edit > Convert > To Current Swift Syntax.

Hope you understand why we made this choice.
